library/core/src/num/f32.rs

```diff
@@ -628,6 +628,8 @@ impl f32 {
     ///
     /// assert!(abs_difference <= f32::EPSILON);
     /// ```
+    #[must_use = "this returns the result of the operation, \
+                  without modifying the original"]
```

Comment:
```suggestion
    #[must_use = "this returns the result of the operation, without modifying the original"]
```

And likewise for the rest; these all appear to take less than 100 characters.

Reply:
ah, I copied the format from other places in the file, not seeing that it was within the 100 char limit, that's much nicer